StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -213,7 +202,73 @@ class EmbeddedPaymentMethodsView: UIView {
     }
 
     func didTapAccessoryButton() {
-        // TODO(porter)
+        delegate?.presentSavedPaymentMethods(selectedSavedPaymentMethod: selection?.savedPaymentMethod)
+    }
+
+    func updateSavedPaymentMethodRow(_ savedPaymentMethod: STPPaymentMethod?,
+                                     isSelected: Bool,
+                                     accessoryType: RowButton.RightAccessoryButton.AccessoryType?) {
+        guard let previousSavedPaymentMethodButton = self.savedPaymentMethodButton,
+              let viewIndex = stackView.arrangedSubviews.firstIndex(of: previousSavedPaymentMethodButton) else {
+            stpAssertionFailure("Function supports only when there is an existing savedPaymentMethodButton")
+            return
+        }
+
+        if let savedPaymentMethod {
+            // Replace saved payment method button at same index
+            let updatedSavedPaymentMethodButton = makeSavedPaymentMethodButton(savedPaymentMethod: savedPaymentMethod,
+                                                                               savedPaymentMethodAccessoryType: accessoryType)
+            // If the previous item was as selected spm or the user explicitly selected a spm, then auto-select saved pm
+            if selection?.isSaved ?? false || isSelected {
```

Comment:
Why isn't this just `if isSelected {`?  

Is it intentional that this method can select the PM even when you pass `isSelected=false`?

Reply:
It is intentional, assuming that this is the expected behavior

Consider the scenario:
1. User has a single CBC enabled card
2. User selects the single CBC card, putting in a selected state
3. User taps edit (UpdateCardViewController is presented) 
4. User Switches from Visa -> CB (or CB->VISA), then taps update, UpdateCardViewController is dismissed

Expected Result:
The updated single CBC card should preserve the selected state (which happened in step 2). 

If we don't check `selection?.isSaved`, then the updated card will be unselected.

Reply:
This seems like a code smell; `isSelected` doesn't do what you'd expect as a caller (it does not necessarily select the PM).

Maybe we could either:
- Rename `isSelected` to something that better matches its behavior.
- Rework so that `isSelected` has the expected behavior. Maybe the caller can pass `isSelected: true` in the scenario above?